### PR TITLE
Update system requirements

### PR DESCRIPTION
**Pull request by settermjd**
This change removes PHP 5.x from the list of supported and recommended PHP versions.

💁‍♂️ - **Only backport to 10.2**

### CODE REVIEW

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 1]
```diff
@@ -38,14 +38,14 @@ we officially recommend and support:
 | * Apache 2.4 with xref:installation/manual_installation.adoc#multi-processing-module-mpm[`prefork` and `mod_php`]
 
 | PHP Runtime
-| * 5.6, 7.0, 7.1, and 7.2
```
Comment (pmaier1) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: 7.0 is still supported.

Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: core 10.2 release drops PHP 5.6 but still supports PHP 7.0

**Comment by settermjd** [comment]
Thanks for catching that oversight.

**Comment by settermjd** [comment]
Is PHP 7.0 still supported? Because https://github.com/owncloud/core/blob/master/composer.json#L35 shows that the minimum PHP version allowed is 7.1. Am I looking at the wrong file?

**Comment by pmaier1** [comment]
> Is PHP 7.0 still supported?

It is. Probably you need to look at stable10 branch.

**Comment by settermjd** [comment]
Fair enough. Just checked and agree.

**Comment by settermjd** [comment]
@pmaier1, 7.0 has been re-added.

**Pull request merged**